packages/plugin-ext/src/common/plugin-api-rpc.ts

```diff
@@ -1157,11 +1157,11 @@ export interface DecorationOptions {
 }
 
 export interface TextEditorsMain {
-    // $tryShowTextDocument(resource: UriComponents, options: TextDocumentShowOptions): Promise<string>;
+    $tryShowTextDocument(uri: UriComponents, options?: TextDocumentShowOptions): Promise<void>;
     $registerTextEditorDecorationType(key: string, options: DecorationRenderOptions): void;
     $removeTextEditorDecorationType(key: string): void;
     // $tryShowEditor(id: string, position: EditorPosition): Promise<void>;
```

Comment:
Should we implement / delete this variant?

Reply:
I will remove it. Looking at the arguments the idea might have been to show/split an existing editor to a tab bar. 
I think this can be achieved with `tryShowTextDocument` as well